src/components/CompositeBar/Item/Item.tsx

```diff
@@ -155,6 +156,12 @@ export const Item: React.FC<ItemInnerProps> = (props) => {
     };
 
     const makeNode = ({icon: iconEl, title: titleEl}: MakeItemParams) => {
+        const iconNode = makeIconNode(iconEl);
+
+        if (bringForward) {
+            highlightedNode = iconNode;
```

Comment:
The function `makeNode` has become a function with a side effect. I suggest leaving the variable `highlightedNode` definition in its place, but correcting the logic of its initialization (consider `ItemWrapper`).